gym_xiangqi/constants.py

```diff
@@ -4,6 +4,12 @@
 """
 
 
+""" PYGAME """
+winWidth = 800
+winHeight = 577
```

Comment:
Just wondering if there is any specific reason these are in lower case.

Reply:
> Just wondering if there is any specific reason these are in lower case.

nope. i thought someone else defined it like that. Maybe it was me. Fixed!